### PR TITLE
[IMP] update 8.0

### PR DESCRIPTION
**Pull request by ventilooo**

### CODE REVIEW

**Comment by ghost** [comment]
:+1: LGTM

**Review by ghost — APPROVED** [review]

File: 8.0/Dockerfile [hunk 1]
```diff
@@ -45,31 +49,32 @@ RUN set -x; \
...
+    cd /fake_buildout && \
+    curl -o bootstrap-buildout.py -SL https://bootstrap.pypa.io/bootstrap-buildout.py && \
+    curl -o /tmp/requirements.txt -SL https://github.com/odoo/odoo/raw/${VERSION}/requirements.txt && \
+    sed -i 's/pyparsing==1.5.7/pyparsing==2.1.10/' /tmp/requirements.txt && \
+    awk -F== '{print "     " $1}' /tmp/requirements.txt >> buildout.cfg && \
+    awk -F== '{print "     " $1}' /tmp/requirements-extra.txt >> buildout.cfg && \
+    sed 's|==| = |' /tmp/requirements.txt > buildout-requirements.txt && \
+    sed 's|==| = |' /tmp/requirements-extra.txt > buildout-requirements-extra.txt && \
+    echo "\n[versions]\n" >> buildout.cfg && \
+    curl -o get-pip.py -SL https://bootstrap.pypa.io/get-pip.py && \
```
Comment (joaoalf) [inline 8.0/Dockerfile]: @ventilooo Duplicated line. See L.69

File: 8.0/Dockerfile [hunk 2]
```diff
@@ -45,31 +49,32 @@ RUN set -x; \
...
+    cat buildout-requirements.txt >> buildout.cfg && \
+    cat buildout-requirements-extra.txt >> buildout.cfg && \
+    python bootstrap-buildout.py && \
+    bin/buildout && \
+    ls -la /opt/${USER}/.buildout/eggs && \
```
Comment (joaoalf) [inline 8.0/Dockerfile]: @ventilooo This line is for debug purpose.

File: 8.0/requirements-extra.txt [hunk 3]
```diff
@@ -3,3 +3,11 @@ coverage==4.2
 geoip==1.3.2
 pymssql==2.1.3
 SQLAlchemy==1.1.4
+GeoIP==1.3.2
```
Comment (joaoalf) [inline 8.0/requirements-extra.txt]: @ventilooo Duplicated

**Comment by joaoalf** [comment]
@ventilooo I tested and it works. I just need you to fix  the small problems pointed in the comments above to merge.

**Comment by ventilooo** [comment]
@joaoalf Done !

**Comment by joaoalf** [comment]
+1

**Pull request merged**